streams/src/main/java/org/apache/kafka/streams/errors/internals/FailedProcessingException.java

```diff
@@ -24,13 +24,20 @@
  */
 public class FailedProcessingException extends StreamsException {
     private static final long serialVersionUID = 1L;
+    private final String failedProcessorNodeName;
 
-    public FailedProcessingException(final String errorMessage, final Exception exception) {
+    public FailedProcessingException(final String errorMessage, final String failedProcessorNodeName, final Exception exception) {
         super(errorMessage, exception);
+        this.failedProcessorNodeName = failedProcessorNodeName;
     }
 
-    public FailedProcessingException(final Exception exception) {
+    public FailedProcessingException(final String failedProcessorNodeName, final Exception exception) {
         // we need to explicitly set `message` to `null` here
         super(null, exception);
+        this.failedProcessorNodeName = failedProcessorNodeName;
+    }
+
+    public String getFailedProcessorNodeName() {
```

Comment:
We do not use the `get`-prefix with getters. Please rename this method to `failedProcessorNodeName()`.

Reply:
@cadonna Fixed